### PR TITLE
Commerce 4667 license test

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:stable

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci.test:stable

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:stable

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:stable

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:stable

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by jpince** [comment]
@andrea-ale-sbarra `ci:test:relevant` runs relevant AND stable tests. In most cases, you should only run `ci:test:relevant` and `ci:test:sf`. Suggested uses below:

`ci:test:relevant` - I'm done with my code and want to run all relevant tests (e.g. stable tests + commerce tests)
`ci:test:stable` - I'm not done with my code and want to run basic tests (build, startup, basic smoke/integration tests)
`ci:test:bundle` - I'm not done with my code and just want to check to make sure it compiles on CI

@marco-leo @pierpinna please share this with the rest of the team. Extra ci:test runs slows down PR results for everyone, please use CI resources wisely :smile:

**Comment by marco-leo** [comment]
ci:test:stable

**Comment by jpince** [comment]
@marco-leo lots of configuration-related exceptions after upgrade (see below). Looks related to the changes. There are other errors as well, please read the logs to figure out what is wrong. I think the additional stable test run will show the same results.
``` 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsFactoryImpl(713)] : The setConfigurationBeanDeclaration method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsFactoryImpl(713)] : The setConfigurationBeanDeclaration method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsFactoryImpl(713)] : The setConfigurationBeanDeclaration method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsFactoryImpl(713)] : The setConfigurationBeanDeclaration method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsFactoryImpl(713)] : The setConfigurationBeanDeclaration method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsFactoryImpl(713)] : The setConfigurationBeanDeclaration method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsFactoryImpl(713)] : The setConfigurationPidMapping method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsFactoryImpl(713)] : The setConfigurationPidMapping method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsFactoryImpl(713)] : The setConfigurationPidMapping method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsFactoryImpl(713)] : The setConfigurationPidMapping method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsFactoryImpl(713)] : The setConfigurationPidMapping method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsLocatorHelperImpl(714)] : The setConfigurationPidMapping method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsLocatorHelperImpl(714)] : The setConfigurationPidMapping method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsLocatorHelperImpl(714)] : The setConfigurationPidMapping method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsLocatorHelperImpl(714)] : The setConfigurationPidMapping method has thrown an exception 
     [exec]     LIFERAY_ERROR: bundle com.liferay.portal.configuration.settings:5.0.11 (237)[com.liferay.portal.configuration.settings.internal.SettingsLocatorHelperImpl(714)] : The setConfigurationPidMapping method has thrown an exception 
```

**Comment by jpince** [comment]
If you need help with analysis, please ask in #d-quality-assurance

File: modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/servlet/taglib/ui/CPDefinitionScreenNavigationConstants.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.commerce.product.definitions.web.servlet.taglib.ui;
+package com.liferay.commerce.product.servlet.taglib.ui;
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/servlet/taglib/ui/CPDefinitionScreenNavigationConstants.java:15]: @andrea-ale-sbarra we should actually change this to `package com.liferay.commerce.product.constants` or `package com.liferay.commerce.product.servlet.taglib.ui.constants` would work too. Just got to have constants be the ending path.

File: modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/servlet/taglib/ui/CPInstanceScreenNavigationConstants.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.commerce.product.definitions.web.servlet.taglib.ui;
+package com.liferay.commerce.product.servlet.taglib.ui;
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/servlet/taglib/ui/CPInstanceScreenNavigationConstants.java:15]: same here, change this to `package com.liferay.commerce.product.constants` or `package com.liferay.commerce.product.servlet.taglib.ui.constants` would work too. Just got to have constants be the ending path.

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci.test:commerce

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by yichenroy** [comment]
ci:test:relevant

Reresting, the failure on `functional-upgrade-tomcat90-mysql57-jdk8_stable/6` seems to be a CI issue.

**Comment by jpince** [comment]
@marco-leo see @yichenroy 's comment above. This pull should be good to forward to Brian.

**Comment by marco-leo** [comment]
@andrea-ale-sbarra @dsitu the pull looks good. Please send a follow up to remove the sf suppression rule that now is satisfied, as well we have to document those as breaking changes in the proper file.  Thanks

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**